modules/sdk-core/src/bitgo/keychain/iKeychains.ts

```diff
@@ -64,12 +64,12 @@ export interface AddKeychainOptions {
   provider?: string;
   reqId?: IRequestTracer;
   krsSpecific?: any;
-  keyShares?: KeyShare[];
+  keyShares?: ApiKeyShare[];
   userGPGPublicKey?: string;
   backupGPGPublicKey?: string;
 }
 
-interface KeyShare {
+export interface ApiKeyShare {
```

Comment:
We need to export this so that we can use the type in the `modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts` file. Exporting with the name as is was causing a conflict with another exported type of the same name somewhere else.